src/parsley/field.js

```diff
@@ -132,6 +133,8 @@ Field.prototype = {
     if ('undefined' === typeof value || null === value)
       value = this.getValue();
 
+    console.log('validate' + value)
```

Comment:
Please remove

Reply:
I have updated the commit now to remove the console-statement as well as revert the dist and doc-folders.